From 74c37f4af3bb65346f52de298bee8d3dd1f89b17 Mon Sep 17 00:00:00 2001
From: Antonio Ospite <ospite@studenti.unina.it>
Date: Thu, 7 Mar 2013 15:58:16 +0100
Subject: [PATCH] debian/patches/: #0007 fix getting system path
X-Face: z*RaLf`X<@C75u6Ig9}{oW$H;1_\2t5)({*|jhM<pyWR#k60!#=#>/Vb;]yA5<GWI5`6u&+
 ;6b'@y|8w"wB;4/e!7wYYrcqdJFY,~%Gk_4]cq$Ei/7<j&N3ah(m`ku?pX.&+~:_/wC~dwn^)MizBG
 !pE^+iDQQ1yC6^,)YDKkxDd!T>\I~93>J<_`<4)A{':UrE

Don't use version string when getting the system path.

Patch 0007 fixes this for the installation part, but is incomplete as it
does not cover the runtime query of the system path.

Fix also a typo in the description of patch 0007.

Closes: #702515
---
 ...7-do_not_use_version_number_in_system_path.patch |   19 ++++++++++++++++---
 1 file changed, 16 insertions(+), 3 deletions(-)

diff --git a/debian/patches/0007-do_not_use_version_number_in_system_path.patch b/debian/patches/0007-do_not_use_version_number_in_system_path.patch
index d08a0cd..75f4f3b 100644
--- a/debian/patches/0007-do_not_use_version_number_in_system_path.patch
+++ b/debian/patches/0007-do_not_use_version_number_in_system_path.patch
@@ -2,12 +2,13 @@ From: "Matteo F. Vescovi" <mfv.debian@gmail.com>
 Date: Fri, 11 Jan 2013 15:37:53 +0100
 Subject: do_not_use_version_number_in_system_path
 
-Global installation expects system patch to be
+Global installation expects system path to be
 something like /usr/share/blender/scripts/
 without version numbers.
 ---
- source/creator/CMakeLists.txt | 2 +-
- 1 file changed, 1 insertion(+), 1 deletion(-)
+ intern/ghost/intern/GHOST_SystemPathsX11.cpp | 3 +++
+ source/creator/CMakeLists.txt                | 2 +-
+ 2 files changed, 4 insertions(+), 1 deletion(-)
 
 diff --git a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
 index 7ea1dd3..6553ce3 100644
@@ -22,3 +23,15 @@ index 7ea1dd3..6553ce3 100644
  		endif()
  	endif()
  
+--- a/intern/ghost/intern/GHOST_SystemPathsX11.cpp
++++ b/intern/ghost/intern/GHOST_SystemPathsX11.cpp
+@@ -60,6 +60,9 @@ GHOST_SystemPathsX11::~GHOST_SystemPaths
+ 
+ const GHOST_TUns8 *GHOST_SystemPathsX11::getSystemDir(int, const char *versionstr) const
+ {
++	/* XXX On Debian ignore versionstr when building the system path */
++	versionstr = "";
++
+ 	/* no prefix assumes a portable build which only uses bundled scripts */
+ 	if (static_path) {
+ 		static char system_path[PATH_MAX];
-- 
1.7.10.4

